pan/types.pan

```diff
@@ -816,6 +816,24 @@ function is_email = {
     true;
 };
 
+@documentation{
+    desc = require a comma-separated list of email addresses
+}
+function is_email_list = {
+    if (ARGC != 1 || !is_string(ARGV[0]))
+        error("usage: is_email_list(string)");
```

Comment:
You're right about the indent check being wrong, I'll make a note to have a look at it.
Would this be an acceptable work-around for now?
```suggestion
    if (ARGC != 1 || !is_string(ARGV[0])) error("usage: is_email_list(string)");
```

Reply:
I think I would rather do:
```
    if (ARGC != 1 || !is_string(ARGV[0])) {
         error("usage: is_email_list(string)");
    };
```